test/I.js

```diff
@@ -18,4 +20,12 @@ describe('I', function() {
        {'@@type': 'my-package/Foreign'});
   });
 
+  it('is idempotent', function() {
+    ok(properties.idempotent(S.I), true);
```

Comment:
I think you mean `ok(...)` rather than `ok(..., true)`. I would prefer `eq(..., true)`, though. I appreciate the simplicity of using a single assertion function, and the added correctness guarantee it provides (if `...` does not evaluate to a Boolean value our error will be brought to our attention if we use `eq(..., true)` but not if we use `ok(...)`, which is equivalent to `eq(Boolean(...), true)`).